package.json

```diff
@@ -31,7 +31,8 @@
     "permissionSets": [
       {
         "permissionName": "settings.remote-storage.enabled",
-        "displayName": "Settings (Remote storages): display list of settings pages",
+        "displayName": "Settings (Remote storages): Module is enabled.",
+        "visible": false,
```

Comment:
hmm, if it's not visible, what is the reason to have it? you can include "settings.enabled" directly, if I'm not missing something

Reply:
o, it's a default settings permission, I see